### PR TITLE
Improve configuration

### PR DESCRIPTION
**Pull request by roschaefer**
I had a moment of 'a-ha!' after reading through: https://github.com/lorenwest/node-config/wiki/Configuration-Files

The goal of this PR is to improve the configuration and remove redundancy. And I really dislike magic generation of configuration files. That was confusing me a while ago.

- [x] helpful error message if `mongo` server is not running
- [x] run `yarn run clear` before `mocha`

### CODE REVIEW

**Comment by roschaefer** [comment]
I'm currently testing maildev. Is it possible that this is never `true`: https://github.com/Human-Connection/API/blob/fcf5353e35e5054e54c1d93b67515ae9e3629749/server/services/admin/admin.class.js#L141

File: package.json [hunk 1]
```diff
@@ -33,26 +33,18 @@
...
-    "test:mongo": "npm run eslint && cross-env NODE_ENV=test && nyc --reporter=lcovonly --reporter=text-summary npm run mocha:mongo",
+    "mongo": "mongod --dbpath data --quiet",
+    "mongo:win": "mongod --dbpath /data/db --quiet",
+    "test": "yarn run mocha && yarn run cucumber",
+    "test:coverage": "nyc --reporter=lcovonly --reporter=text-summary yarn run test",
     "eslint": "eslint server/. test/. features/. --config .eslintrc.json",
-    "start": "concurrently 'mongod' 'wait-on tcp:27017 && cross-env NODE_ENV=production node server/'",
-    "dev:debug": "npm run clear && concurrently '$npm_package_config_mongoDev &>/dev/null' 'wait-on tcp:27017 && cross-env DEBUG=feathers && cross-env NODE_ENV=development nodemon --inspect server/'",
-    "dev": "npm run clear && concurrently '$npm_package_config_mongoDev &>/dev/null' 'wait-on tcp:27017 && cross-env DEBUG=feathers && cross-env NODE_ENV=development nodemon server/'",
-    "dev:local": "sh scripts/run-local.sh",
```
Comment (appinteractive) [inline package.json]: could you please re-add the dev:local command for mobile development?

**Comment by appinteractive** [comment]
good work!

**Comment by appinteractive** [comment]
> I'm currently testing maildev. Is it possible that this is never true:
```sendEmail: data.sendInviteEmails === true```

This line is set to true by an api call while importing invited users via thre WebApp. You can choose to send those mails or not in the process.

File: package.json (hunk 1 above)
Comment (roschaefer) [reply package.json]: The script uses `ifconfig` to set some environment variables with the local IP address. E.g. I don't have `ifconfig` installed and the script fails silently. I find it much more convenient to set `host` in `config/local.json` to the local ip and just run `yarn run dev`. Can we keep `scripts/run-local.sh` deleted, please?

Comment (roschaefer) [reply package.json]: I can add some instructions in `README.md` if you like.

Comment (appinteractive) [reply package.json]: You missed the point of that script I think. It is getting your host ip to then start the app in your local network so you can access it from your mobile phone. It’s way more convenient then to write your ip into the local.config

But for me it’s fine to put it into the readme.

**Pull request merged**